.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.idea
```

Comment:
this sould be in your global git config
check https://git-scm.com/docs/gitignore section about excludesfile